specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -5189,6 +5203,48 @@
       },
       "description": "Response from a List SynonymMaps request. If successful, it includes the full definitions of all synonym maps."
     },
+    "EncryptionKey": {
+      "properties": {
+        "keyVaultKeyName": {
+          "type": "string",
+          "description": "The name of the key, from your Azure KeyVault, you wish to use to protect your data. "
```

Comment:
How about "The name of a key that you create in Azure KeyVault. This key will be used to encrypt your data at rest."

Reply:
I like Bruce's suggestion.  Azure Key Vault is 3 words.